qemu/tests/numa_stress.py

```diff
@@ -107,7 +105,9 @@ def run(test, params, env):
     if not os.path.isdir(tmpfs_path):
         os.mkdir(tmpfs_path)
 
-    numa_node_malloc = -1
+    test_mem = int(params.get("mem")) - 2048
+    stress_args = "--cpu 4 --io 4 --vm 2 --vm-bytes %sM" % int(test_mem / 2)
+    stress_test = utils_test.VMStress(vm, "stress", params, stress_args=stress_args)
```

Comment:
Should move this line into below for loop, to initialize a new instance each time loading stress.

Reply:
But I just want to run stress command in each round test, it's duplicate to download/install stress before run it every time.  I guess it's better to complement a function to only run stress if it has been installed. Thanks

Reply:
@huangyum Yeah, I absolutely understand this requirement, and I hate this too. But it just hardly to find a common way to check if the application is installed since `Stress()` is designed not only for "stress" app, say, for iozone it will be totally different(not installed to /usr/bin), so this is a compromise fomula to execute the whole process in each loop.
And also, the original  `autotest_control.run()` will do the same, to download and install the application each time being called. So VMStress is just a totally same replacement of that.
Let me know if you insist to ask change `Stress()`, then we might rewrite most part of it...

Reply:
@vivianQizhu Thanks for the info.  I'm fine to keep it is for now and have updated. Please help review again.